Tools/AP_Bootloader/board_types.txt

```diff
@@ -392,6 +392,9 @@ AP_HW_CUAV-7-NANO                    7000
 # IDs 7100-7109 reserved for VIEWPRO
 AP_HW_VUAV-V7pro                     7100
 
+# IDs 7200-7209 reserved for X-MAV
+AP_HW_X-MAV-AP-H743V2                7200
```

Comment:
Please don't leave this huge gap in IDs.  Fill in one of our *other* huge gaps.

Reply:
ok, can i use 7110-7119 ?

Reply:
Already changed AP_HW_X-MAV-AP-H743V2 id to 1168, and filled the gap now.